api/maven-api-settings/src/main/mdo/settings.mdo

```diff
@@ -459,17 +459,19 @@
         The <code>&lt;proxy&gt;</code> element contains informations required to a proxy settings.
         ]]></description>
       <fields>
-        <field>
-          <name>active</name>
+        <field xml.tagName="active">
+          <name>activeString</name>
           <version>1.0.0+</version>
           <required>false</required>
           <defaultValue>true</defaultValue>
           <description>
             <![CDATA[
```

Comment:
CDATA not needed here

Reply:
All descriptions in this file are enclosed in CDATA blocks, so while I don't mind removing those, I think it belongs in a separate PR.  I suppose the idea was to add CDATA blocks to all human readable texts and java code.

Reply:
It's syntax sugar, unnecessary here though and just excess code. It can be fixed one by one, or not, as you like. There's no need to change everything at once.